torchvision/io/_video_opt.py

```diff
@@ -24,21 +24,20 @@
 # simple class for torch scripting
 # the complex Fraction class from fractions module is not scriptable
 class Timebase(object):
-    __annotations__ = {"numerator": int, "denominator": int}
-    __slots__ = ["numerator", "denominator"]
+    __annotations__: Dict[str, Type[int]] = {"numerator": int, "denominator": int}
+    __slots__: List[str] = ["numerator", "denominator"]
```

Comment:
Are these necessary? Can't `mypy` infer them automatically?

Reply:
That might be true, but I thought it would be better for future contributors that class attributes would have specified types. Do you think we should remove this? (going for the minimal so that mypy is happy I mean)

Reply:
> Do you think we should remove this?

IMO, yes. Since this is not a public interface, we should let `mypy` infer whatever it can.